code/AssetLib/3DS/3DSLoader.cpp

```diff
@@ -449,7 +449,7 @@ void Discreet3DSImporter::ParseChunk(const char *name, unsigned int num) {
         // Read the lense angle
         camera->mHorizontalFOV = AI_DEG_TO_RAD(stream->GetF4());
         if (camera->mHorizontalFOV < 0.001f) {
-            camera->mHorizontalFOV = AI_DEG_TO_RAD(45.f);
+            camera->mHorizontalFOV = float(AI_DEG_TO_RAD(45.f));
```

Comment:
Please use ai_real instead

Reply:
mHorizontalFOV and configMaxAngle are floats by definition and should not be changed to double. The AI_DEG_TO_RAD return ai_real, so we need to convert to float.

Reply:
I am fine with that.